### PR TITLE
Best commit ever

### PR DESCRIPTION
**Pull request by flashlee**
Initial project setup with help of CRA
D3 and apollo clients installed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# IllWatch UI
```
Comment (maybelinot) [inline README.md]: lower `i`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Client web app for IllWatch project
+
+## Usage
+
+### Installing
```
Comment (maybelinot) [inline README.md]: ~Installing~ &rarr; Installation ?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# IllWatch UI
+
+Client web app for IllWatch project
```
Comment (maybelinot) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+### Installing 
+Install all dependencies by running
+
+    yarn 
+or
+
+    yarn install
+
+### Development server
+For development purposes you can start an app by command
```
Comment (maybelinot) [inline README.md]: ~an app~ &rarr; the app?
~by command~ &rarr;  "with command" or "by running"?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+    yarn start
+By default development server is started on `localhost:3000`
```
Comment (maybelinot) [inline README.md]: ~is started~ &rarr;  starts?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+### Build app
+For bundling app into static files for production
```
Comment (maybelinot) [inline README.md]: ~For bundling app~ &rarr;  To bundle the app ?

**Review by maybelinot — CHANGES_REQUESTED** [review]

File: package.json [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-boost": "^0.1.16",
+    "d3": "^5.7.0",
+    "graphql": "^14.0.2",
+    "react": "^16.5.1",
+    "react-apollo": "^2.1.11",
+    "react-dom": "^16.5.1",
+    "react-scripts": "1.1.5"
```
Comment (alienalien13) [inline package.json]: Please remove `^` everywhere
Why:
https://bytearcher.com/articles/semver-explained-why-theres-a-caret-in-my-package-json/

**Review by alienalien13 — CHANGES_REQUESTED** [review]

**Comment by alienalien13** [comment]
🎉 🎉 🎉

File: package.json [hunk 8]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "ill-watch-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-boost": "^0.1.16",
```
Comment (alienalien13) [inline package.json]: 👍

File: README.md (hunk 2 above)
Comment (flashlee) [reply README.md]: Without context I think it doesn't really matter which word to use.
see e.g. https://github.com/npm/cli#installing-on-cygwin

**Comment by flashlee** [comment]
@maybelinot @alienalien13  :eyes: ?

File: README.md (hunk 2 above)
Comment (maybelinot) [reply README.md]: I'd prefer to have a noun there, it's really weird to see a verb in this particular case and form..
@alienalien13 WDYT?

Comment (maybelinot) [reply README.md]: Installing on smth would be better, but just `Installing` with no context doesn't sound at all

**Review by alienalien13 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (alienalien13) [reply README.md]: What about just ‘Install’?

Comment (maybelinot) [reply README.md]: I just want to be consistent, right now we have different form in each paragraph:
* Development Server
* Installing 
* Build app

I'm totally fine with either all verbs:
* Develop
* Install
* Build

or nouns
* Develompent
* Installation
* Build

Comment (alienalien13) [reply README.md]: Good point.
Nouns will work for me as well as verbs.

Comment (flashlee) [reply README.md]: @maybelinot edited

**Review by maybelinot — APPROVED** [review]

**Comment by maybelinot** [comment]
@flashlee @alienalien13 Congrats guys, first PR behind!

**Pull request merged**